pkg/asset/installconfig/kubevirt/utils.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubevirt
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+var (
+	kubeConfigEnvName         = "KUBECONFIG"
+	kubeConfigDefaultFilename = filepath.Join(os.Getenv("HOME"), ".kube", "config")
+)
+
+// LoadKubeConfigContent returns the kubeconfig file content
+func LoadKubeConfigContent() ([]byte, error) {
+	kubeConfigFilename := os.Getenv(kubeConfigEnvName)
+	// Fallback to default kubeconfig file location if no env variable set
+	if kubeConfigFilename == "" {
+		kubeConfigFilename = kubeConfigDefaultFilename
+	}
+
+	return ioutil.ReadFile(kubeConfigFilename)
+}
+
+// WaitForDeletionComplete waits until the resource is not found, getFunc returns error
+func WaitForDeletionComplete(
```

Comment:
is this the client.go only function?

Reply:
terrible comment :) I meant that if this function only use is from the client.go then it make sense to move it there. Also should it be public?

Reply:
This is util function giving an option to validate resource deletion is completed (after any delete function was called in client.go)
Its not part of client.go, because client.go should be only a wrapper, without additional logic